common/buildings/00_duchy_capital_buildings.txt

```diff
@@ -341,6 +341,9 @@ siege_works_02 = {
 	character_modifier = {
```

Comment:
Check `light_cavalry_` in `00_martial_lifestyle_modifiers.txt`, these aren't adapted.